doc-content/enterprise-only/openshift/operator-createapp-proc.adoc

```diff
@@ -0,0 +1,154 @@
+[id='operator-createapp-proc']
+= Deploying a {PRODUCT} environment using the operator
+
+After subscribing to the operator, you can _create an application_ to deploy an environment. 
+
+.Procedure
+
+. Enter your project in the OpenShift Web cluster console. 
+. In the OpenShift Web console navigation panel, select *Operators* and then *Subscriptions*.
+. Click the name of the subscription that containt `businessautomation`. Information about this subscription is displayed.
+. Under the *Installed version* heading, click the version name of the subscription. An overview of the subscription is displayed.
+. Click the *Create KieApp* button. A YAML source is displayed.
+. Set the *name* field to an application name that is unique in the project.
+. Set the *environment* field to the required environment type. Each of the types is a default deployment pattern. You can modify the pattern by editing this YAML source; you can also modify the deployment after it is completed. The following types are available:
+
+** `rhpam-trial`: Includes {CENTRAL} and a {KIE_SERVER}. You can set it up quickly and use it to evaluate or demonstrate developing and running assets. However, the environment does not use any persistent storage, and any work you do in the environment is not saved.
```

Comment:
Maybe for consistency with the others: "A trial environment for ... . This environment includes {CENTRAL} and ..."